askup/templates/askup/user_profile.html

```diff
@@ -167,13 +169,15 @@ <h2>Incorrect answers</h2>
                                 My subscriptions
                         </a>
                         {% endif %}
+                        {% if is_student %}
```

Comment:
Why don't move `Rank list` closer to the `My/Other questions` and fiar with one `if student` block?